app/routers/profile.py

```diff
@@ -196,3 +212,41 @@ async def update(
     finally:
         url = router.url_path_for("profile")
         return RedirectResponse(url=url, status_code=HTTP_302_FOUND)
+
+
+@router.get("/restore_events")
+async def restore_events(request: Request, session=Depends(get_db)):
+    # TODO: Add user.id instead 1
```

Comment:
why not using current user as you did below?

Reply:
Thanks